package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/cmc-common-frontend",
-  "version": "1.21.2",
+  "version": "1.21.3",
```

Comment:
version needs an update if this is still wanted

Reply:
It is not priority but translation should be done inside to follow our approach. Bumped version.